packages/core/src/shipping/create-shipping-strategy-registry.ts

```diff
@@ -54,6 +76,19 @@ export default function createShippingStrategyRegistry(
             ),
     );
 
+    registry.register(
```

Comment:
Can you please update a test for this too.

Reply:
done